packages/pretty-format/src/plugins/HTMLElement.js

```diff
@@ -60,7 +61,7 @@ const print = (
 ) => {
   let result = colors.tag.open + '<';
   const elementName = element.constructor
```

Comment:
`tagName` defaults to all uppercase, which felt odd to me -- should I leave it that way or is the `.toLowerCase()` call okay here?